### PR TITLE
Feature/adding vuetify

### PR DESCRIPTION
**Pull request by pani-tka**
Added Vuetify and changed source components to vuetify-components.

### CODE REVIEW

File: src/components/todo-item.vue [hunk 1]
```diff
@@ -38,15 +67,24 @@ export default {
...
   validations: {
     editingValue: {
-      required,
       maxLength: maxLength(25)
     }
   },
+  computed: {
+    validationError() {
+      const errors = [];
+      if (!this.$v.editingValue.$dirty) return errors;
```
Comment (hollote) [inline src/components/todo-item.vue]: brackets?

File: src/components/todo-list.vue [hunk 2]
```diff
@@ -32,6 +42,12 @@ import TodoItem from "./todo-item";
 export default {
   name: "TodoList",
   components: {TodoItem},
+  data() {
+    return {
+      filters: ['inProgress', 'completed', 'all'],
+      currentFilter: "inProgress",
```
Comment (hollote) [inline src/components/todo-list.vue]: move to `computed` from vuex

File: src/components/add-new-todo.vue [hunk 3]
```diff
@@ -31,67 +41,28 @@ export default {
...
     newTodo: {
-      required,
       maxLength: maxLength(25)
     }
   },
+  computed: {
+    validationError() {
+      const errors = [];
+      if (!this.$v.newTodo.$dirty) return errors;
+      !this.$v.newTodo.maxLength && errors.push(`This field must have less than
```
[the remark this answers is not in the document]
Comment (pani-tka) [reply src/components/add-new-todo.vue]: Fixed.

File: src/App.vue [hunk 4]
```diff
@@ -1,17 +1,32 @@
...
+                    indeterminate
+                  ></v-progress-circular>
+                </v-layout>
+              </v-container>
+            </v-dialog>
+          </v-layout>
+        </div>
+        <v-card min-height="300" rised v-else>
+          <v-toolbar-title class="text-center pa-12">YOUR TODO-APP</v-toolbar-title>
+          <AddNewTodo></AddNewTodo>
```
[the remark this answers is not in the document]
Comment (pani-tka) [reply src/App.vue]: Fixed.

File: src/App.vue [hunk 5]
```diff
@@ -1,17 +1,32 @@
...
+                    :width="7"
+                    color="light-green darken-3"
+                    indeterminate
+                  ></v-progress-circular>
+                </v-layout>
+              </v-container>
+            </v-dialog>
+          </v-layout>
+        </div>
+        <v-card min-height="300" rised v-else>
```
[the remark this answers is not in the document]
Comment (pani-tka) [reply src/App.vue]: Removed rised, but v-card left.  If remove v-card, then we need to replace it with a div or another container(?) to save v-else condition.

[the remark this answers is not in the document]
Comment (hollote) [reply src/App.vue]: use `div` instead

File: src/App.vue [hunk 6]
```diff
@@ -60,3 +60,11 @@ export default {
 }
 </script>
 
+<style scoped>
```
Comment (hollote) [inline src/App.vue]: please not create custom styles if you can use vuetify instead

**Pull request merged**